examples/new_member.py

```diff
@@ -1,22 +1,22 @@
-# This example requires the 'members' privileged intent to use the Member converter.
-
 import discord
 
+intents = discord.Intents.default()
+intents.members = True  # This intent requires "Server Member Intent" to be enabled at https://discord.com/developers
```

Comment:
This isn't exactly the full link to the Developers Dashboard

Reply:
It redirects there

Reply:
Nevermind about this one, I think my brain isn't working